skeleton/controller/acp_controller.php.twig

```diff
@@ -64,7 +68,7 @@ class acp_controller
 	public function display_options()
 	{
 		// Add our common language file
-		$this->lang->add_lang('common', '{{ EXTENSION.vendor_name }}/{{ EXTENSION.extension_name }}');
+		$this->{{ LANGUAGE.service }}->{{ LANGUAGE.function }}({% if LANGUAGE.s_32 %}'common', {% endif %}'{{ EXTENSION.vendor_name }}/{{ EXTENSION.extension_name }}'{% if not LANGUAGE.s_32 %}, 'common'{% endif %});
```

Comment:
This may be more verbose but probably easier to understand if like:
```
{% if LANGUAGE.service == 'language' %}
$this->{{ LANGUAGE.service }}->{{ LANGUAGE.function }}('common', '{{ EXTENSION.vendor_name }}/{{ EXTENSION.extension_name }});
{% else %}
$this->{{ LANGUAGE.service }}->{{ LANGUAGE.function }}('{{ EXTENSION.vendor_name }}/{{ EXTENSION.extension_name }}, 'common');
```